packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -242,8 +242,8 @@ export const getZeroDevProjectIdByBlockchain = logInputOutput((chain: EVMBlockch
     return zeroDevProjectId;
 }, "getZeroDevProjectIdByBlockchain");
 
-export const getViemNetwork = logInputOutput((networkName: EVMBlockchainIncludingTestnet) => {
```

Comment:
The `logInputOutput` here was causing type issues so dropping. If you think it's important I'll improve the typing on `logInputOutput`